strawberry_django/__init__.py

```diff
@@ -21,6 +22,22 @@
 from .utils import fields
 
 
+try:
+    from django.contrib.gis import geos  # noqa: F401
+
+    from .fields.types import (  # noqa: F401
+        LinearRing,
+        LineString,
+        MultiLineString,
+        MultiPoint,
+        MultiPolygon,
+        Point,
+        Polygon,
+    )
+except ImproperlyConfigured:
+    # If gdal is not available, skip.
+    pass
```

Comment:
```suggestion
try:
    from .fields.types import (  # noqa: F401
        LinearRing,
        LineString,
        MultiLineString,
        MultiPoint,
        MultiPolygon,
        Point,
        Polygon,
    )
except ImportError:
    # If gdal is not available, skip.
    pass
```

Reply:
This won't work as it relies on `from django.contrib.gis import geos` to determine if geos libraries are installed